source/code/include/scxcorelib/scxatomic.h

```diff
@@ -113,6 +115,22 @@ __inline static bool scx_atomic_decrement_test(scx_atomic_t* v)
 #endif
 
 #elif defined(linux)
+
+// Use intrinsics for IBM powerPC
+#if defined(ppc)
+
+static __inline__ void scx_atomic_increment(scx_atomic_t* v)
+{
+    ++(*v);
```

Comment:
Um, is this REALLY atomic? Isn't there some sort of real "atomic increment" routine on PPC? We have atomic unit tests; do you know if our PPC platform VM was built with multiple CPUs for proper atomic tests?